specs/passenger_spec.rb

```diff
@@ -75,4 +76,67 @@
       end
     end
   end
+
+  describe "total spent method" do
+    # Arrange----------------
+    before do
```

Comment:
I see the same missing tests as for the methods in `Driver`: 0 trips and an incomplete trip.